chain/rust/src/transaction/mod.rs

```diff
@@ -375,6 +376,10 @@ impl TransactionBody {
             encodings: None,
         }
     }
+
+    pub fn hash(&self) -> [u8; 32] {
+        blake2b256(&self.to_cbor_bytes())
```

Comment:
`[u8; u32]` is more general but less consistent than `TransactionHash`. I guess on the rust side there's an `Into` impl so it's okay.